### PR TITLE
ACSF-TOOLS: Support Drush 10.x.

### PR DESCRIPTION
**Pull request by jayakrishnanj**
Removed deprecated drush 9.x functions.

### CODE REVIEW

**Comment by vbouchet31** [comment]
I created a new branch for 10.x (from 9.x-dev) and updated your PR to use this branch instead.

**Comment by vbouchet31** [comment]
I tested the patch and it seems drush acsf-tools-ml works as expected.

drush acsf-tools-dump is not working. Using verbose mode, it shows that drush sql-dump is invoked with both result-file (expected) and result-folder (not expected).
I think there is some change around $options management (probably $options = Drush::redispatchOptions(); is returning even $options which was unset).

**Comment by jayakrishnanj** [comment]
Great I will look into this.

File: AcsfToolsCommands.php [hunk 1]
```diff
@@ -260,29 +245,33 @@ public function dbDump(array $options = ['result-folder' => '~/drush-backups', '
...
 
+    // Identify target folder.
+    $result_folder = $options['result-folder'];
+    $current_date = date("Ymd");
+    // Folder based on current date.
+    $backup_result_folder = $result_folder . '/' . $current_date;
     // If dump directory does not exist.
-    if (!file_exists($options['result-folder'])) {
-      $directory_message = sprintf('Dump directory "%s" does not exist. Do you want to create this directory?', $options['result-folder']);
+    if(!file_exists($backup_result_folder)){
```
Comment (sdelbosc) [inline AcsfToolsCommands.php:254]: missing spaces before after brackets

**Comment by sdelbosc** [comment]
Merging as is as better than what we have on that branch anyways

**Pull request merged**